packages/graphics/kmscube/package.mk

```diff
@@ -2,14 +2,14 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="kmscube"
-PKG_VERSION="98f31bf"
-PKG_SHA256="78b52b9e606f0d3444e10ea2ed7c0c03a87f1ad2ef99e35036551395faade041"
+PKG_VERSION="26326be53e30da9c101075fda081d38ea9ec758d"
+PKG_SHA256="ce96a78edd37387058a81070950c993853f427959cfafc15cfc5f78e9f2e9b07"
 PKG_LICENSE="GPL"
-PKG_SITE="https://cgit.freedesktop.org/mesa/kmscube"
-PKG_URL="https://cgit.freedesktop.org/mesa/kmscube/snapshot/$PKG_VERSION.tar.xz"
+PKG_SITE="https://gitlab.freedesktop.org/mesa/kmscube"
+PKG_URL="https://gitlab.freedesktop.org/mesa/kmscube/-/archive/master/kmscube-${PKG_VERSION}.tar.gz"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="Example KMS/GBM/EGL application"
-PKG_TOOLCHAIN="autotools"
+PKG_TOOLCHAIN="auto"
```

Comment:
you should just be able to drop this line completely

Reply:
done (and tested)